### PR TITLE
Create Event “fil-singapore-alongside-token2049-2024”

### PR DESCRIPTION
**Pull request by oliviasigler**
Automatically generated by Decap CMS

### CODE REVIEW

File: public/assets/images/fil-singapore-vertical-light.png
Comment (mirhamasala) [inline public/assets/images/fil-singapore-vertical-light.png:1]: @oliviasigler - Please check the hints in the Content Management System and follow them! 

For images, it says: `"Use https://squoosh.app/ to compress images before uploading. Choose the WebP format and resize the image to 2000px width."`

File: src/content/events/fil-singapore-alongside-token2049-2024.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  twitter:
+    title: summary
+    card: summary
+  description: >-
+    ​Hosted by MetaPals and supported by Filecoin Foundation, FILSG: Web3
+    Wonderland is the marquee event of Token 2049, bringing together some of the
+    sharpest minds and most innovative projects across Web3 and AI.
+
+
+    ​Enjoy the incredible view of Singapore's skyline and F1 track at the highest food forest in the world - 1-Arden!
```
Comment (mirhamasala) [inline src/content/events/fil-singapore-alongside-token2049-2024.md:23]: @oliviasigler - The hint in the CMS says" `"To generate meta title and description, copy and paste the entry content into ChatGPT 4.0, then use the prompt: 'Generate a meta title and a meta description under 160 characters for the following content: [paste your content here].'"`

Comment (mirhamasala) [reply src/content/events/fil-singapore-alongside-token2049-2024.md:23]: Is this supposed to be the main `description` or the SEO description?

**Review by mirhamasala — CHANGES_REQUESTED** [review]
@oliviasigler - Please check the comments and adapt accordingly. Let me know if anything is unclear. 🙏🏼

File: src/content/events/fil-singapore-alongside-token2049-2024.md (hunk 1 above)
Comment (oliviasigler) [reply src/content/events/fil-singapore-alongside-token2049-2024.md:23]: Join FILSG: Web3 Wonderland at 1-Arden with stunning Singapore views. Discover top Web3 and AI innovations alongside TOKEN2049.

File: public/assets/images/fil-singapore-vertical-light.png
Comment (oliviasigler) [reply public/assets/images/fil-singapore-vertical-light.png:1]: Hi! How do I upload the new photo it won't let me :/

Sorry if this is a silly question!

**Comment by oliviasigler** [comment]
> @oliviasigler - Please check the comments and adapt accordingly. Let me know if anything is unclear. 🙏🏼

responded!

**Comment by mirhamasala** [comment]
Closing in favor of https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/493

**Pull request closed without merging**